### PR TITLE
Updated Example in event.md

### PR DESCRIPTION
**Pull request by kylearoberts**
One of the examples needed to be updated to match with what we currently are sending. There is also one minor correction to the wording of one definition.

### CODE REVIEW

**Comment by ksigler7** [comment]
@kylearoberts the build broke because your updates aren't valid json. I took the example you added, and fixed the JSON, and came up with this:

```
[  
   {  
      "email":"john.doe@sendgrid.com",
      "timestamp":1337197600,
      "smtp-id":"4FB4041F.6080505@sendgrid.com",
      "sg_event_id":"sendgrid_internal_event_id",
      "sg_message_id":"sendgrid_internal_message_id",
      "event":"processed"
   },
   {  
      "email":"john.doe@sendgrid.com",
      "timestamp":1337966815,
      "ip":"X.XX.XXX.XX",
      "sg_event_id":"sendgrid_internal_event_id",
      "url":"https://sendgrid.com",
      "sg_message_id":"sendgrid_internal_message_id",
      "useragent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/61.0.3163.100 Safari/537.36",
      "event":"click"
   },
   {  
      "ip":"X.XX.XXX.XX",
      "sg_user_id":"XXXXXX",
      "sg_event_id":"sendgrid_internal_event_id",
      "sg_message_id":"sendgrid_internal_message_id",
      "useragent":"Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/61.0.3163.100 Safari/537.36",
      "event":"group_unsubscribe",
      "email":"john.doe@sendgrid.com",
      "timestamp":1337969592,
      "asm_group_id":42
   }
]
```
Does this match what we are currently sending?

File: source/API_Reference/Webhooks/event.md [hunk 1]
```diff
@@ -62,26 +62,33 @@ Event POST Example
...
     "timestamp": 1337966815,
-    "category": "newuser",
-    "event": "click",
-    "url": "https://sendgrid.com"
+    "ip":"X.XX.XXX.XX",
+    "sg_event_id":"sendgrid_internal_event_id",
+    "url":"https://sendgrid.com",
+    "sg_message_id":"sendgrid_internal_message_id",
+    "useragent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/61.0.3163.100 Safari/537.36",
+    "event":"click",
```
Comment (ksigler7) [inline source/API_Reference/Webhooks/event.md]: get rid of comma

File: source/API_Reference/Webhooks/event.md [hunk 2]
```diff
@@ -62,26 +62,33 @@ Event POST Example
...
+
```
Comment (ksigler7) [inline source/API_Reference/Webhooks/event.md]: get rid of extra space

**Comment by thinkingserious** [comment]
Hello @kylearoberts,<br /><br/>Thanks again for the PR!<br />                                              <br />It's <a href="https://sendgrid.com/blog/announcing-hacktoberfest-2017">HACKTOBERFEST</a>! We want to show our appreciation by sending                                              you some special Hacktoberfest swag. If you have not already, could you please fill out                                              <a href="https://dx.sendgrid.com/swag">this form</a> so we can send it to you? Thanks!                                             <br /><br />Team SendGrid DX

**Pull request merged**